### ISSUE TITLE
Use of `Any` as a value expression should be flagged as error

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#6144]
When `Any` is used outside of a type annotation in a regular value expression, it should be treated as a normal `object` for purposes of type checking. The following should generate an error, but does not currently:

```python
from typing import Any

def func() -> int:
    return Any # Should generate an error
```

**Comment by erictraut** [comment]
This is included in pyright 1.1.332, which I just published. It will also be included in a future release of pylance.
